Jenkinsfile

```diff
@@ -2,12 +2,24 @@
 node {
   stage('checkout sources') {
         // You should change this to be the appropriate thing
-        git url: 'https://github.com/jschmersal-cscc/special-topics-labs-quality'
+        git url: 'https://github.com/Felixda1/special-topics-labs-ci.git'
   }
 
   stage('Build') {
-    // you should build this repo with a maven build step here
-    echo "hello"
+    withMaven (maven: 'maven3') {
+              sh "mvn package"
+            }
+  }
+  stage('Test') {
+              steps {
+                  sh './src/test/java/edu/cscc/topics/quality/unit check'
+              }
+          }
+      }
+      post {
+          always {
+              junit 'src/test/results.xml'
```

Comment:
Watch out here... to see what the path should be, run `mvn package` from the root directory of your repo.  Where do the test results land?  (hint - maven puts its build output, including test results, somewhere under `target`).

Reply:
Didn't realize that was actually an important location. Only thought the one for where the tests were was important. Seems to be putting them in surefire-reports so updating that now. Is this important because jenkins only picks up from a specific location or something?